content/docs/central-dash/registration-flow.md

```diff
@@ -0,0 +1,25 @@
++++
```

Comment:
Let's put all the central dashboard docs into one place, in the **components** section of the docs. So:

Move this page from `content/docs/central-dash/registration-flow.md` to `content/docs/components/central-dash/registration-flow.md`.

Reply:
Done